awx/ui/test/e2e/fixtures.js

```diff
@@ -299,7 +299,19 @@ const getAuditor = (namespace = session) => getOrganization(namespace)
 
 const getUser = (namespace = session) => getOrganization(namespace)
     .then(organization => getOrCreate(`/organizations/${organization.id}/users/`, {
-        username: `user-${uuid().substr(0, 8)}`,
+        username: `${namespace}-user-${uuid().substr(0, 8)}`,
```

Comment:
we need to keep the `namespace` out of the username, as some tests (e.g: xss) will fail due to server-side validation of usernames.

Reply:
I think there might be validation issues with that breaking other tests(?) but I can investigate its effects first thing tomorrow.